lib/smack/SmackInstGenerator.cpp

```diff
@@ -652,13 +662,6 @@ void SmackInstGenerator::visitCallInst(llvm::CallInst& ci) {
     WARN("ignoring llvm.debug call.");
     emit(Stmt::skip());
 
-  } else if (name.find("llvm.expect.") != std::string::npos) {
```

Comment:
Move handling of debug intrinsics above into `processIntrinsicCall` as well.

Reply:
> Move handling of debug intrinsics above into `processIntrinsicCall` as well.

Good catch. Done.